WordPress/wporg-mu-plugins
# Tools: Try adding infrastructure for PHPUnit tests

I think it would be helpful to add unit tests for the global-font changes in #297, but we need the infrastructure in place first. This forks out the phpunit code from #204, and rebases it all to the current `trunk`.

To test:

- `yarn` to install the wp-env dep
- `composer update` to install the new composer deps
- `yarn test:php` to run the tests (currently just the demo test)

The actions run on this PR should also include a unit-test action.
